js/noms/src/list.js

```diff
@@ -155,7 +155,7 @@ export default class List<T: Value> extends Collection<IndexedSequence<any>> {
   /**
    * Returns a new JS array with the same values as list.
    */
-  // $FlowIssue
+  // $FlowIssue: ???
```

Comment:
it should be
```javascript
toJS(start: number = 0, end: (number | undefined) = undefined) {
  if (end === undefined) {
    end = this.length;
  }
  ...
}
```
it doesn't like `this` inside a default parameter list, which makes sense in a way.